app/src/main/java/com/amaze/filemanager/crashreport/ErrorActivity.java

```diff
@@ -496,5 +502,9 @@ public void writeToParcel(final Parcel dest, final int flags) {
       dest.writeString(this.request);
       dest.writeInt(this.message);
     }
+
+    public static int comparePackageInfo(String packageName) {
+      return packageName.indexOf("com.amaze.filemanager");
```

Comment:
How is this going to work on debug package?

Reply:
```kotlin
"com.amaze.filemanager.debug".indexOf("com.amaze.filemanager") == 0
```